src/sinol_make/helpers/package_util.py

```diff
@@ -49,3 +49,37 @@ def get_executable_path(solution: str) -> str:
     Returns path to compiled executable for given solution.
     """
     return os.path.join(os.getcwd(), 'cache', 'executables', get_executable(solution))
+
+
+def get_time_limit(test_path, config):
+    """
+    Returns time limit for given test.
+    """
+    test_id = extract_test_id(test_path)
+    test_group = get_group(test_path)
+
+    if "time_limits" in config:
+        if test_id in config["time_limits"]:
+            return config["time_limits"][test_id]
+        elif int(test_group) in config["time_limits"]:
+            return config["time_limits"][int(test_group)]
+        elif str(test_group) in config["time_limits"]:
+            return config["time_limits"][str(test_group)]
+    return config["time_limit"]
```

Comment:
That looks weird, is there really no better way to write this code?

Reply:
I changed it a bit